nsls2forge_utils/auto_tick.py

```diff
@@ -151,26 +149,14 @@ def bot_pr_body(self, feedstock_ctx, channel_org='nsls2forge'):
         )
     )
     # Statement here
-    template = (
-        "|{name}|{new_version}|[![Anaconda-Server Badge]"
-        "(https://img.shields.io/conda/vn/{channel_org}/{name}.svg)]"
-        "(https://anaconda.org/{channel_org}/{name})|\n"
+    curr_pkg_name = feedstock_ctx.package_name
+    pred.insert(0, curr_pkg_name)
+    body += (
+        "\n\nHere is a list of all the pending dependencies (and their "
+        "versions) for this repo. The first package is the current feedstock package. "
+        "Please double check all dependencies before merging.\n\n"
```

Comment:
Wow, it even exceeded what I requested :). It's really good to have a list of dependencies as well.